.github/workflows/issue-board-onboard.yml

```
@@ -0,0 +1,526 @@
+              }
+              await setSingleSelect(itemId, STATUS_FIELD, STATUS_IN_TRIAGE);
+              const sprintField = await field(SPRINT_FIELD);
+              const sprint = currentIteration(sprintField.iterations, new Date());
+              if (sprint) {
+                await setIteration(itemId, SPRINT_FIELD, sprint);
+              } else {
+                core.warning(
+                  `no current iteration on field ${SPRINT_FIELD}; leaving Sprint unset`);
+              }
+            }
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)